Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/nodejs-16:latest AS build
+FROM registry.redhat.io/ubi9/nodejs-18:latest AS web-builder
```

Comment:
If we change the image alias, we need to update the COPY step accordingly

Reply:
That makes sense. Updated in latest commit.